packages/client/src/pages/Agent/AgentShow.re

```diff
@@ -1,56 +1,80 @@
-open Utils;
-open Foretold__GraphQL;
-open Css;
-
-let block = style([marginBottom(`em(1.0))]);
-let notFound = "Agent not found" |> ste |> E.React.inH3;
-
-/* TODO: Improve this bit */
-let agentSection = (e: Queries.Agent.agent) =>
-  switch (e) {
-  | {bot: Some(r)} =>
-    <>
-      {r.name |> ste |> E.React.inH2}
-      {
-        r.description
-        |> E.O.fmap(r => r |> ste |> E.React.inH3)
-        |> E.O.React.defaultNull
-      }
-      {
-        r.competitorType
-        |> Context.Primary.CompetitorType.toString
-        |> ste
-        |> E.React.inH3
-      }
-    </>
-  | {user: Some(r)} => r.name |> ste |> E.React.inH1
-  | _ => notFound
+module ReducerConfig = {
```

Comment:
I think we should actually rename this page from AgentShow -> AgentMeasurements, which is more accurate.

Reply:
Yes, it has a sense.

Reply:
Fixed.